File: ServerWApp/Models/Restorant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerWApp.Models
{
    public class Restorant
    {
        public int RestorantId { get; set; }
        public string? RestorantName { get; set; }

        public string? RestorantAdres { get; set; }

        public string? RestSahibiFullName { get; set; }

        public string? RestAciklama { get; set; }
        public string? RestSahibiEMail { get; set; }
        public string? RestorantEMail { get; set; }
        public string? RestSahibiPhone {get; set;}
        public string? Dakika {get; set;}
        public string? Parola {get; set;}
        public string? Image {get; set;}
        public int KitchenId {get;set;}
        public string? KitchenName {get;set;}
        public virtual Kitchen? Kitchen{ get; set;}

        
    }
}

File: ng-app/src/app/models/restorant.ts
export class Model{
  restorants: Array<Restorant>;
}

export class Restorant{
  restorantId: number;
  restorantName: string;
  restorantAdres: string;
  restSahibiFullName: string;
  restSahibiEMail: string;
  restSahibiPhone: string;
  restorantEMail:string;
  parola:string;
  kitchenId:number;
  kitchenName: string;



    constructor(Id: number, restname: string, adres:string, SahibiFullName: string, SahibiEMail: string,SahibiPhone: string,RestEmail:string,Sifre:string,Image:string,KitchenId:number,kitchenName:string){
      this.restorantId = Id;
      this.restorantName = restname;
      this.restorantAdres = adres;
      this.restSahibiFullName = SahibiFullName;
      this.restSahibiEMail = SahibiEMail;
      this.restSahibiPhone = SahibiPhone;
      this.restorantEMail= RestEmail;
      this.parola=Sifre;
      this.kitchenId= KitchenId;
      this.kitchenName=kitchenName;
    }
}

// export interface Restaurant{
//   [x: string]: any;
//   id: number;
//   name: string;
//   imageUrl: string;
//   isActive: boolean;
//   dk: string;
//   restorantId: number;
//   restCategoryId: number;

// }
